stable/spring-cloud-data-flow/README.md

```diff
@@ -8,7 +8,7 @@ Pipelines consist of [Spring Boot](http://projects.spring.io/spring-boot/) apps,
 This chart will provision a fully functional and fully featured Spring Cloud Data Flow installation
 that can deploy and manage data processing pipelines in the cluster that it is deployed to.
 
-MySQL is used as the data store for Spring Cloud Data Flow state and either RabbitMQ or Kafka is used as the messaging layer for streaming apps to communicate with one another.
+MySQL and RabbitMQ are used as the default data store for Spring Cloud Data Flow state and messaging layer for streaming apps to communicate with one another respectively.
```

Comment:
i'd rather not call RabbitMQ a "data store" here. can this change be reverted?

Reply:
It's respectively, meaning MySQL->data store, RabbitMQ->Messaging layer. To make it clearer, I will make it 2 sentences.